store/migration/dev/20220514000002__extension.sql

```diff
@@ -0,0 +1,27 @@
+-- extension stores the extensions for a particular database.
+-- data is synced periodically from the instance.
+CREATE TABLE extension (
```

Comment:
Call it pg_extension as extension might be reserved for our own extension system

Reply:
Done. Renamed to db_extension.